pkg/kubelet/status/status_manager_test.go

```diff
@@ -583,6 +583,29 @@ func TestStaticPod(t *testing.T) {
 	})
 }
 
+func TestTerminatePod(t *testing.T) {
+	syncer := newTestManager(&fake.Clientset{})
+	testPod := getTestPod()
```

Comment:
Could you set a fake status here with `Running` phase, and running container? And add some comment about why we add this test.